src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php

```diff
@@ -625,4 +585,82 @@ private function cloneAndClearTokens(Tokens $tokens, int $start, int $end): arra
 
         return $clone;
     }
+
+    /**
+     * @return array|array<string, array{positive: string, negative: false|string, argument_count?: int, swap_arguments?: true}|true>
```

Comment:
```suggestion
     * @return array<string, array{positive: string, negative: false|string, argument_count?: int, swap_arguments?: true}|true>
```

Reply:
I went hardcore mode and defined the array shape